public/pages/QueryInsights/QueryInsights.tsx

```diff
@@ -70,16 +72,63 @@ const QueryInsights = ({
   };
 
   const cols: Array<EuiBasicTableColumn<any>> = [
+    {
+      name: QUERY_HASHCODE,
+      render: (query: any) => {
+        return (
+          <span>
+        <EuiLink onClick={() => {
+          const route = query.group_by === 'SIMILARITY' ? `/query-group-details/${hash(query)}` : `/query-details/${hash(query)}`;
+          history.push(route);
+        }}>
+          {query.query_hashcode || '-'}
```

Comment:
We should probably use query_id for this if displaying single query: https://github.com/opensearch-project/query-insights/issues/159